modules/sdk/src/main/scala/org/tessellation/sdk/infrastructure/consensus/ConsensusState.scala

```diff
@@ -18,9 +20,20 @@ case class ConsensusState[Key, Artifact](
   statusUpdatedAt: FiniteDuration
 )
 
-@derive(eqv, show)
+@derive(eqv)
 sealed trait ConsensusStatus[Artifact]
 
+object ConsensusStatus {
```

Comment:
Thanks for fixing this, we need to limit logging to essential only info and this wasn't essential  👍